addons/hr_contract/report/hr_contract_history_report_views.xml

```diff
@@ -199,12 +199,4 @@
             </kanban>
         </field>
     </record>
-
-    <menuitem
-        id="hr_menu_contract_history"
-        action="hr_contract_history_view_list_action"
-        parent="hr.menu_hr_employee_payroll"
-        name="Contracts"
-        sequence="4"
-        groups="hr_contract.group_hr_contract_manager"/>
```

Comment:
are the hr_contract_history model and views still accessible from somewhere? As I understand from the specs, everything should be removed

Reply:
yes Yannick told me to not remove the history model because it was used somewhere in payroll for optimizations i think

Reply:
Ok! Thanks for the insight! :+1: